egs/babel_multilang/s5/local/nnet3/run_common_langs.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# This script uses to generated MFCC+pitch features for input language L.
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # If true, it regenerates alignments. 
+speed_perturb=true
+use_pitch=true      # If true, it generates pitch features and combine it with 40dim MFCC.
+pitch_conf=conf/pitch.conf # Configuration used for pitch extraction.
+use_pitch_plp=false # If true, it generated plp+pitch to be used in regenerating alignments.
+
+[ ! -f ./conf/common_vars.sh ] && echo 'the file conf/common_vars.sh does not exist!' && exit 1
+
+. conf/common_vars.sh || exit 1;
+
+[ -f local.conf ] && . ./local.conf
+
+. ./utils/parse_options.sh
+
+lang=$1
+
+# perturbed data preparation
+train_set=train
+if [ "$speed_perturb" == "true" ]; then
+  if [ $stage -le 1 ]; then
+    #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
+    # _sp stands for speed-perturbed
+    for datadir in train; do
+      ./utils/data/perturb_data_dir_speed_3way.sh data/$lang/${datadir} data/$lang/${datadir}_sp
+
+      # Extract Plp+pitch feature for perturbed data.
```

Comment:
actually this feature extraction is inside speed-perturb condition, where you need to regenerate alignment for perturbed data.